changelogs/fragments/validate-modules-deprecated_aliases.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- ansible-test validate-modules - Correctly validate the ``deprecated_aliases`` argument spec option
```

Comment:
I don't think this needs a changelog fragment, since the bad test has only been merged for a couple of days, and hasn't been released so far.

Reply:
Wouldn't we want to backport this to stable-2.9 so that's actually validating `deprecated_aliases`? While it fixes the issue in devel we currently don't validate `deprecated_aliases` in stable-2.9 at all.

Reply:
You would have to either backport #65747 as well, or completely rewrite your PR, since there was no schema validation for argument_spec until three days ago. And if you also backport #65747, I would argue that this doesn't need a changelog fragment since it is a bugfix for #65747.

Reply:
(Also, I'm not sure whether such extensive new tests should be backported, since I would consider them new features. But maybe it's different for ansible-test?)

Reply:
Ah fair enough makes sense.

Reply:
I've added the question whether such tests should be backported to the public project meeting agenda (https://github.com/ansible/community/issues/519#issuecomment-580457957).